spec/request/authentication/standard_authentication.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+# Note these specs MUST be run in order
+RSpec.describe 'authenticating loa3 user', type: :request, order: :defined do
+  OUTBOUND_CASSETTE = 'complex_interaction/external_interactions'
+  Episode = Struct.new(:method, :uri, :body, :headers, :recorded_at, :response)
+
+  EPISODES = begin
+    inbound_cassette_path = 'spec/support/vcr_cassettes/complex_interaction/internal_interactions.yml'
+    YAML.load(File.read(inbound_cassette_path))['http_interactions'].map do |interaction|
+      req = interaction['request']
+      req['uri'] = URI.parse(req['uri'])
+      req['recorded_at'] = Time.zone.parse(interaction['recorded_at'].to_s).to_datetime
+      req['headers'] = Hash[req['headers'].map { |k, v| [k, v.first] }]
+      Episode.new(*req.values, interaction['response'])
+    end
+  end
+
+  it 'does the tests', :aggregate_failures, :skip_mvi, :skip_emis do
```

Comment:
`does the tests` is not very useful.  Might I recommend, 
```
Rspec.describe 'Authentication' ...
```
followed with,
```
it 'authenticates at loa3' do ...
```

Reply:
Agree with these comments. I started working on a gem: 

https://github.com/saneshark/dual_deck

But I'm not going to work on that on company time for now. Right now I just needed a basic baseline to have some confidence mucking around with stuff. I'll continue to work on the gem on the weekend on my own time and eventually replace.